readme.md

```diff
@@ -51,41 +51,99 @@ Complete the following exercises to practice using SQL.
 
 ###Order
 1. Find all subjects sorted by subject
+
+  > SELECT subject FROM subjects ORDER BY subjects ASC;
 2. Find all subjects sorted by location
 
+  > SELECT subject FROM subjects ORDER BY location ASC;
+
 ###Where
 1. Find the book "Little Women"
+
+  > SELECT * FROM books WHERE title = 'Little Women';
 2. Find all books containing the word "Python"
+
+  > SELECT * FROM books WHERE title ILIKE '%python%';
 3. Find all subjects with the location "Main St" sort them by subject
 
+  > SELECT * FROM subjects WHERE location = 'Main St' ORDER BY subject ASC;
+
 
 ###Joins
 
 * Find all books about Computers list ONLY book title
+  > SELECT title FROM books
+  > INNER JOIN subjects
+  > ON books.subject_id = subjects.id
+  > WHERE subject = 'Computers';
+
 * Find all books and display ONLY
 	* Book title
 	* Author's first name
 	* Author's last name
 	* Book subject
+    > SELECT books.title, authors.first_name, authors.last_name, subjects.subject 
+    > FROM books
+    > INNER JOIN authors
+    > ON books.author_id = authors.id
+    > INNER JOIN subjects
+    > ON books.subject_id = subjects.id;
 * Find all books that are listed in the stock table
 	* Sort them by retail price (most expensive first)
 	* Display ONLY: title and price
+
+    > SELECT books.title, stock.retail FROM stock
+    > INNER JOIN editions
+    > ON stock.isbn = editions.isbn
+    > INNER JOIN books
+    > ON editions.book_id = books.id
+    > ORDER BY stock.retail DESC;
+
 * Find the book "Dune" and display ONLY
 	* Book title
 	* ISBN number
 	* Publisher name
 	* Retail price
+
+    > SELECT books.title, editions.isbn, publishers.name, stock.retail FROM books
+    > INNER JOIN editions
+    > ON books.id = editions.book_id
+    > INNER JOIN publishers
+    > ON editions.publisher_id = publishers.id
+    > INNER JOIN stock
+    > ON stock.isbn = editions.isbn
+    > WHERE books.title = 'Dune';
+
 * Find all shipments sorted by ship date display ONLY:
 	* Customer first name
 	* Customer last name
 	* ship date
 	* book title
 
+    > SELECT books.title, customers.first_name, customers.last_name, 
+    > shipments.ship_date FROM shipments
+    > LEFT JOIN customers
+    > ON shipments.customer_id = customers.id
+    > LEFT JOIN editions
+    > ON editions.isbn = shipments.isbn
+    > LEFT JOIN books
+    > ON editions.book_id = books.id
+    > ORDER BY shipments.ship_date ASC;
+
```

Comment:
great, here is a slightly different method too - select customers.first_name, customers.last_name, shipments.ship_date, books.title from shipments join editions on shipments.isbn =editions.isbn join customers on shipments.customer_id = customers.id join books on editions.book_id = books.id order by shipments.ship_date;